cmd/chartmuseum/main.go

```diff
@@ -95,7 +95,7 @@ func cliHandler(c *cli.Context) {
 		UseStatefiles:          !conf.GetBool("disablestatefiles"),
 		AllowOverwrite:         conf.GetBool("allowoverwrite"),
 		AllowForceOverwrite:    !conf.GetBool("disableforceoverwrite"),
-		EnableMetrics:          !conf.GetBool("disablemetrics"),
+		EnableMetrics:          !conf.GetBool("enablemetrics"),
```

Comment:
Should be `conf.GetBool("enablemetrics")` ?

Reply:
i thought `conf.GetBool("enablemetrics")` will enable the metrics by default. we are trying to disable it

Reply:
You can use `conf.GetBool("enablemetrics")` and default the value to `false` to achieve the same thing, it just makes things a bit cleaner.

Reply:
Gotcha!